operator/api/v1alpha1/keptntaskdefinition_types.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// KeptnTaskDefinitionSpec defines the desired state of KeptnTaskDefinition
+type KeptnTaskDefinitionSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// Foo is an example field of KeptnTaskDefinition. Edit keptntaskdefinition_types.go to remove/update
+	Function FunctionSpec `json:"function,omitempty"`
+}
+
+type FunctionSpec struct {
+	FunctionReference  FunctionReference  `json:"functionRef,omitempty"`
+	InlineReference    InlineReference    `json:"inlineRef,omitempty"`
```

Comment:
i think the `Ref` suffix can be omitted for the inline field, since we are not referencing any other object here

Reply:
done